Jenkinsfile

```diff
@@ -0,0 +1,137 @@
+stage 'Parallel Build'
+
+def variants = [
+    'clang.debug.unity',
+    'clang.debug.nounity',
+    'gcc.debug.unity',
+    'gcc.debug.nounity',
+    'gcc.release.unity',
+    'coverage'] as String[]
+
+// create a map of all builds
+// that we want to run. The map
+// is string keys and node{} object values
+def builds = [:]
+for (int index = 0; index < variants.size(); index++) {
+    def bldtype = variants[index]
+    builds[bldtype] = {
+        node('rippled-dev') {
```

Comment:
The example for `parallel` uses different node names
https://jenkins.io/doc/book/pipeline/jenkinsfile/#advanced-scripted-pipeline
Would it be useful to use the build variant as the node name?